src/router/middlewares/setTenantId.ts

```diff
@@ -48,6 +48,92 @@ const getTenantIdFromAudClaim = (audClaim: any, baseUrl: string): string | undef
     return uniqTenantIds[0];
 };
 
+/**
+ * Evaluates if an all tenants scope matches with the configured one.
+ * @param userIdentity The decoded access token data.
+ * @param fhirConfig The config, which includes multi-tenant setup.
+ * @returns true access granted, false denied
+ */
+function grantAccessForAllTenants(userIdentity: any, fhirConfig: FhirConfig): boolean {
```

Comment:
wrong naming, it does not grant access.  Alternatives: `hasAccessToAllTenants` or `evaluateAccessForAllTenants`